src/app/development/middleware.js

```diff
@@ -68,7 +69,11 @@ async function templatesDisplayGet(req, res) {
 
   if (templateId === "pyi-triage-desktop-download-app") {
     renderOptions.qrCode = await qrCodeHelper.generateQrCodeImageData(
-      SERVICE_URL + "/ipv/app-redirect/" + CONSTANTS.PHONE_TYPES.IPHONE,
+      appDownloadHelper.getAppStoreRedirectUrl(PHONE_TYPES.IPHONE),
+    );
+  } else if (templateId === "pyi-triage-mobile-download-app") {
+    renderOptions.appDownloadUrl = appDownloadHelper.getAppStoreRedirectUrl(
+      PHONE_TYPES.IPHONE,
```

Comment:
In ipv/middleware it's PHONE_TYPES.ANDROID for this one, I assume that was just to test the android link works